pyprep/find_noisy_channels.py

```diff
@@ -144,22 +144,42 @@ def get_bads(self, verbose=False):
             )
         return bads
 
-    def find_all_bads(self, ransac=True):
+    def find_all_bads(self, ransac=True, channel_wise=False, max_chunk_size=None):
         """Call all the functions to detect bad channels.
 
         This function calls all the bad-channel detecting functions.
 
         Parameters
         ----------
-        ransac : bool
-            To detect channels by ransac or not.
+        ransac : bool, optional
+            Whether RANSAC should be for bad channel detection, in addition to
+            the other methods. RANSAC can detect bad channels that other methods
+            are unable to catch, but also slows down noisy channel detection
+            considerably. Defaults to ``True``.
+        channel_wise : bool, optional
+            Whether RANSAC should predict signals for whole chunks of channels
+            at once instead of predicting signals for each RANSAC window
+            individually. Channel-wise RANSAC generally has higher RAM demands
```

Comment:
```suggestion
            Whether RANSAC should predict signals for chunks of channels over the entire
            signal length ("channel-wise RANSAC", see `max_chunk_size` parameter).
            If ``False``, RANSAC will instead predict signals for all channels at once
            but over a number of smaller time windows instead of over the entire
            signal length ("window-wise RANSAC").
            Channel-wise RANSAC generally has higher RAM demands
```

Reply:
Nice and clear, I like it!